Disable altivec on power.

The original compiler flag (-faltivec) was wrong, and after fixing that, the
code doesn't compile with many warnings and errors so fixing it doesn't seem
worthwhile.

https://buildd.debian.org/status/fetch.php?pkg=libfec&arch=ppc64el&ver=1.0%2B20161003-2&stamp=1579519201&raw=0

--- a/configure.in
+++ b/configure.in
@@ -65,10 +65,12 @@ i386|i486|i586|i686)
 	cpu_features.o cpu_mode_x86.o"
 	;;
 powerpc*)
-	ARCH_OPTION="-fno-common -faltivec"
-	MLIBS="viterbi27_av.o viterbi29_av.o viterbi39_av.o viterbi615_av.o \
-	encode_rs_av.o \
-	dotprod_av.o sumsq_av.o peakval_av.o cpu_mode_ppc.o"
+#	ARCH_OPTION="-fno-common -maltivec"
+#	MLIBS="viterbi27_av.o viterbi29_av.o viterbi39_av.o viterbi615_av.o \
+#	encode_rs_av.o \
+#	dotprod_av.o sumsq_av.o peakval_av.o cpu_mode_ppc.o"
+	MLIBS="cpu_mode_unknown.o"
+	CFLAGS+=" -U__vec__ -U__VEC__"
 	;;
 x86_64*)
 	MLIBS="cpu_mode_unknown.o"
